themes/cardinal_service/lib/components/opportunity-row/opportunity_row.html.twig

```diff
@@ -1,7 +1,13 @@
+{% set program = program|render|striptags('<drupal-render-placeholder>')|trim|convert_encoding('UTF-8', 'HTML-ENTITIES') %}
+{% set type_list = type_list|render|striptags('<drupal-render-placeholder>')|trim|convert_encoding('UTF-8', 'HTML-ENTITIES') %}
 {% set time_commitment = time_commitment|render|striptags('<drupal-render-placeholder>')|trim|convert_encoding('UTF-8', 'HTML-ENTITIES') %}
 {% set service_theme = service_theme|render|striptags('<drupal-render-placeholder>')|trim|convert_encoding('UTF-8', 'HTML-ENTITIES') %}
 
-{% if service_theme|render|striptags|trim is not empty %}
+{% if program|render|striptags|trim is not empty and type_list|render|striptags|trim is not empty %}
+  {% set program = (program ~ ', '|t) %}
+{% endif %}
+
+{% if time_commitment|render|striptags|trim is not empty and service_theme|render|striptags|trim is not empty %}
```

Comment:
you're already rendering and doing all that for the time_commitment and service_theme. you can just check if they are not empty here.